apps/backend/src/resources/spec/Resource.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { Resource } from '../Resource';
+
+describe('Resource', () => {
+  it('resourceType should  be defined when initializing a Resouce', () => {
+    class TestResource extends Resource<{}> {}
+    try {
+      const object = new TestResource({});
+    } catch (e) {
+        expect(e instanceof Error).toBeTruthy();
+        expect(e.message).toBe('no resource type specified on TestResource');
```

Comment:
Are we using a custom exceptions messages? We might be able to make the "expect" something more specific if we have standard error messages in the case of an empty Test Resource?